docs/reference/api.yml

```diff
@@ -0,0 +1,3204 @@
+                agribalyse_proxy_food_code:en:
+                  type: string
+                ciqual_food_code:en:
+                  type: string
+            categories_properties_tags:
+              type: array
+              items:
+                type: string
+            categories_tags:
+              type: array
+              items:
+                type: string
+            category_properties:
+              type: object
+              properties:
+                ciqual_food_name:en:
+                  type: string
+                ciqual_food_name:fr:
+                  type: string
+            checkers_tags:
+              type: array
+              items:
+                type: object
+            ciqual_food_name_tags:
+              type: array
+              items:
+                type: string
+            cities_tags:
+              type: array
+              items:
+                type: object
+            code:
+              type: string
+            codes_tags:
+              type: array
+              items:
+                type: string
+            compared_to_category:
+              type: string
+            complete:
+              type: integer
+            completeness:
+              type: number
+            correctors_tags:
+              type: array
+              items:
+                type: string
+            countries:
+              type: string
+            countries_hierarchy:
+              type: array
+              items:
+                type: string
+            countries_lc:
+              type: string
+            countries_tags:
+              type: array
+              items:
+                type: string
+            created_t:
+              type: integer
+            creator:
+              type: string
+            data_quality_bugs_tags:
+              type: array
+              items:
+                type: object
+            data_quality_errors_tags:
+              type: array
+              items:
+                type: object
+            data_quality_info_tags:
+              type: array
+              items:
+                type: string
+            data_quality_tags:
+              type: array
+              items:
+                type: string
+            data_quality_warnings_tags:
+              type: array
+              items:
+                type: string
+            data_sources:
+              type: string
+            data_sources_tags:
+              type: array
+              items:
+                type: string
+            debug_param_sorted_langs:
+              type: array
+              items:
+                type: string
+            ecoscore_data:
+              type: object
+              properties:
+                adjustments:
+                  type: object
+                ecoscore_not_applicable_for_category:
+                  type: string
+                status:
+                  type: string
+            ecoscore_extended_data:
```

Comment:
This is experimental and likely to be removed, I would remove this from the documentation.